spec/exchanges/cex/integration/market_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'spec_helper'
+
+RSpec.describe 'Cex integration specs' do
+  let(:client) { Cryptoexchange::Client.new }
+
+  it 'fetch pairs' do
+    pairs = client.pairs('cex')
+    expect(pairs).not_to be_empty
+
+    pair = pairs.first
+    expect(pair.base).to_not be nil
+    expect(pair.target).to_not be nil
+    expect(pair.market).to eq 'cex'
+  end
+
+  it 'fetch ticker' do
+    usd_btc_pair = Cryptoexchange::Models::MarketPair.new(base: 'USD', target: 'BTC', market: 'cex')
```

Comment:
This needs to be changed as well.

The way to validate this is to look at https://cex.io/api/ticker/BTC/USD
The last price is about 8000
A base USD and target BTC would mean that 1 USD is worth 8000 BTC. 
Something is not right here.

Reply:
base/target position corrected